ccm_web/client/src/models/externalUser.ts

```diff
@@ -42,7 +42,7 @@ interface ExternalUserCreationFailure extends ExternalUserResultBase {
 
 interface ExternalUserInvitationFailure extends ExternalUserResultBase {
   userCreated: true
-  invited: CirrusErrorData
```

Comment:
Removed Reference of Cirrus from Codebase

Note: `server` folder will have reference of it, which will eventually be removed with Python go live time.